src/utils/ADempiere/valueFormat.js

```diff
@@ -114,167 +122,79 @@ export function convertObjectToHasMap({ object }) {
-    currency = getCurrency()
-  }
-  //  Get formatted number
-  return new Intl.NumberFormat(getCountryCode(), {
-    style: 'currency',
-    currency
-  }).format(number)
-}
-
-//  Format Quantity
-export function formatQuantity(number) {
-  if (this.isEmptyValue(number)) {
-    return undefined
-  }
-  if (!Number.isInteger(number)) {
-    return number
-  }
-  return Number.parseFloat(number).toFixed(2)
-  //  Get formatted number
-}
-
-// Format percentage based on Intl library
-export function formatPercent(number) {
-  if (this.isEmptyValue(number)) {
-    return undefined
-  }
-  //  Get formatted number
-  return new Intl.NumberFormat(getCountryCode(), {
-    style: 'percent'
-  }).format(number)
-}
-
-//  Get country code from store
-function getCountryCode() {
-  const languageDefinition = store.getters.getCurrentLanguageDefinition
-  return languageDefinition.languageISO + '-' + languageDefinition.countryCode
-}
+/**
+ * Return a format for field depending of reference for him
+ */
+export function formatField({
+  value,
+  displayedValue,
+  defaultValue,
+  displayType,
+  optionalFormat
+}) {
+  let formattedValue
 
-// Get Default country
-function getCurrency() {
-  const currencyDefinition = store.getters.getCurrency
-  return currencyDefinition.iSOCode
-}
+  //  format
+  switch (displayType) {
+    case (isLookup(displayType) && displayType):
+      if (isEmptyValue(displayedValue) && value === 0) {
+        // TODO: Verify parsedDefaultValue with getDefaultValue
+        formattedValue = defaultValue
+        break
+      }
+      formattedValue = displayedValue
+      break
 
-// Return a format for field depending of reference for him
-export function formatField(value, reference, optionalFormat) {
-  if (isEmptyValue(value)) {
-    return undefined
-  }
-  if (!reference) {
-    return value
-  }
-  //  Format
-  let formattedValue
-  switch (reference) {
     case DATE.id:
-      formattedValue = moment.utc(value).format(getDateFormat({
-        format: optionalFormat
-      }))
+      formattedValue = getDateFormat({
+        isDate: true,
+        isTime: false
+      })
       break
+
     case DATE_PLUS_TIME.id:
-      formattedValue = moment.utc(value).format(getDateFormat({
+      formattedValue = getDateFormat({
+        isDate: true,
         isTime: true
-      }))
+      })
       break
+
     case TIME.id:
-      formattedValue = moment.utc(value).format(getDateFormat({
+      formattedValue = getDateFormat({
+        isDate: false,
         isTime: true
-      }))
+      })
       break
+
     case AMOUNT.id:
-      formattedValue = formatPrice(value)
-      break
     case COSTS_PLUS_PRICES.id:
-      formattedValue = formatPrice(value)
+      formattedValue = formatPrice({
+        value
+      })
       break
+
     case NUMBER.id:
-      formattedValue = formatQuantity(value)
-      break
     case QUANTITY.id:
-      formattedValue = formatQuantity(value)
+    case INTEGER.id:
```

Comment:
Just can see change for source code with the same functionality, please let it without changes